apps/ws/src/GameManager.ts

```diff
@@ -18,12 +18,12 @@ import { Square } from 'chess.js';
 
 export class GameManager {
   private games: Game[];
-  private pendingGameId: string | null;
+  private pendingGameId: { [gameMode: string]: string | null };
```

Comment:
Convert to a record/map
use better types here

Reply:
> Convert to a record/map use better types here

Sir i have changed the types and used Record for the gameMode in my latest commit